src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -335,4 +335,24 @@ private Request matchLoansToRequest(
       .withLoan(loanMap.getOrDefault(request.getItemId(), null));
   }
 
+  public CompletableFuture<Result<MultipleRecords<Loan>>> findOpenLoansByUserIdAndLoanPolicyId(
```

Comment:
Why is the code still using the loan policy ID? Is this a remnant of the old way of doing this?

If not, I don't understand how the flaw in the original has been avoided?